homeassistant/components/insteon/services.yaml

```diff
@@ -2,16 +2,16 @@ add_all_link:
   description: Tells the Insteom Modem (IM) start All-Linking mode. Once the the IM is in All-Linking mode, press the link button on the device to complete All-Linking.
   fields:
     group:
-      description: All-Link group number.
+      description: All-Link group number (0-255).
       example: 1
     mode:
-      description: Linking mode   controller - IM is controller   responder - IM is responder
+      description: "Linking mode:  controller - IM is controller   responder - IM is responder    both - IM is both controller and responder"
```

Comment:
This reads very weird? I suggest using comma to separate the different example options, not multiple spaces.

Reply:
I agree, it's weird layout. I wasn't sure if it was my place to fix it, since it wasn't my code. I'll see if I can clean it up a bit.